docs/advanced-entity-analytics/advanced-entity-analytics-overview.asciidoc

```diff
@@ -8,11 +8,13 @@ Advanced Entity Analytics provides two key capabilities:
 * <<entity-risk-scoring, Entity risk scoring>> 
 * <<advanced-behavioral-detections, Advanced behavioral detections>>
 
+include::ers-req.asciidoc[leveloffset=+1]
 include::entity-risk-scoring.asciidoc[leveloffset=+1]
 include::asset-criticality.asciidoc[leveloffset=+2]
```

Comment:
I'd put the entity risk scoring reqs inside the Entity risk scoring section, since it only applies to that part of Entity Analytics
```suggestion
include::entity-risk-scoring.asciidoc[leveloffset=+1]
include::ers-req.asciidoc[leveloffset=+2]
include::asset-criticality.asciidoc[leveloffset=+2]
```